components/search_bar/search_bar.jsx

```diff
@@ -231,7 +232,7 @@ export default class SearchBar extends React.Component {
             <div className='sidebar-right__table'>
                 <div className='sidebar-collapse__container'>
                     <div
-                        id='sidebarCollapse'
+                        id={this.props.isRHS ? 'rhsSidebarCollapse' : 'sidebarCollapse'}
```

Comment:
Hey @allenlai18, thanks for your contribution :)
Have you considered the Cypress test suite when changing these ids? A lot of them are used by the test suite to simulate browser actions.

Reply:
Yes, so i prefixed all the components that are not visible 'sbr' because all the ones that are not visible are from ```sidebar_right.js``` and the ones that are visible have the same id.  Cypress test suite always refer to the one that is visible. so this does not affect Cypress tests!